pkgs/development/tools/enochecker-test/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, pythonOlder
+, pythonRelaxDepsHook
+
+, certifi
+, charset-normalizer
+, enochecker-core
+, exceptiongroup
+, idna
+, iniconfig
+, jsons
+, packaging
+, pluggy
+, pytest
+, requests
+, tomli
+, typish
+, urllib3
+}:
+
+buildPythonApplication rec {
+  pname = "enochecker-test";
+  version = "0.9.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "enochecker_test";
+    hash = "sha256-M0RTstFePU7O51YVEncVDuuR6F7R8mfdKbO0j7k/o8Q=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+  ];
+
+  pythonRelaxDeps = true;
+
+  propagatedBuildInputs = [
+    certifi
+    charset-normalizer
+    enochecker-core
+    exceptiongroup
+    idna
+    iniconfig
+    jsons
+    packaging
+    pluggy
+    pytest
```

Comment:
```suggestion
```
we don't propagate pytest

Reply:
As far as I understand, `pytest` has to be propagated since it is [imported by `enochecker-test`](https://github.com/enowars/enochecker_test/blob/017df167a74f114b26a9cacd6685ceb3127ef324/enochecker_test/main.py#L7)

Reply:
ah, that rule is only for pytest plugins